modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/util/TopologyUtils.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2023 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.commandline.util;
+
+import org.apache.ignite.internal.client.GridClientCompute;
+import org.apache.ignite.internal.client.GridClientException;
+import org.apache.ignite.internal.client.GridClientNode;
+
+import java.util.Comparator;
+import java.util.Optional;
+import java.util.UUID;
+
+/**
+ * Class provides utility methods to work with nodes in topology:
+ * finding coordinator, filtering client nodes and so on.
+ */
+public final class TopologyUtils {
+    /**
+     * Obtains ID of coordinator node from the topology
+     * visible through {@link GridClientCompute client compute} interface at the moment of method call.
```

Comment:
Should we mention the rule to select the coordinator ('a server node with the least order property')?

Reply:
I addressed this comment, could you please take another look?